### ISSUE TITLE
New theme and suggestion

### ISSUE THREAD

**Issue by smartclash** [nosqlclient/nosqlclient#172]
Hey,

I know you are trying to buy an extended license of INSPINA, but to be true as a developer, i like AdminLTE which is awesome in design and highly documented. It is free and nearly as good as INSPINA. 

AdminLTE is open source and uses bootstrap for design. Probably use that one so you don't have to spend a huge 1000$ for the theme.

Just a suggestion. Please comment on it and let me know :)

EDIT ::

I tried to install this client and when running `meteor npm install` i get this error : 

![selection_002](https://cloud.githubusercontent.com/assets/16922883/18586755/3cbe7abc-7bec-11e6-839a-dcfa40e4cb8a.png)

**Comment by rsercano** [comment]
Hi,

I'll evaluate it and let you know from this issue thanks for the suggestion :)

For your error, obviously you've changed your directory to wrong one as it shows an error on terminal (**No such file or directory**), change this:
`cd mongocliend-master`
to this:
`cd mongoclient-master`

Cheers,
Sercan

**Comment by smartclash** [comment]
Thanks, couldn't know i have made such a silly mistake @rsercano

**Comment by rsercano** [comment]
@smartclash  no problem at all :)

**Comment by rsercano** [comment]
I've had a chance to check your suggested theme @smartclash but since there're a lot of people using mongoclient and INSPINA is way better than it, I rather stay at INSPINA. 

If I reach $700 or something I'm gonna pay rest myself and buy INSPINA.

Thanks & Cheers

**Comment by smartclash** [comment]
Good luck with that. I am just 14, i will try my best to donate you some(atleast a $5 from me). @rsercano

**Comment by rsercano** [comment]
Thank you so much @smartclash

**Comment by zspine** [comment]
@rsercano Yes inspinia definitely a good admin theme and we use it a lot for commercial projects. But are you aware of http://semantic-ui.com/ ? (open source and very clean).

**Comment by rsercano** [comment]
@zspine I didn't know that, thanks for sharing, very soon mongoclient will go institutionalization, and probably will be funded and I'll have a chance to buy INSPINIA. But just in case if it doesn't happen I'll try to switch semantic-ui 

p.s. no worries even if mongoclient goes institutionalization, there'll be no pay model and everything will stay same, apart from it maybe license type can change after new version and nothing else.

**Comment by smartclash** [comment]
@rsercano Is a good friend but they are not fully mobile-friendly. You need to specify different mobile classes to make them friendly. I would recommend you to check out the ground breaker http://materializecss.com/ for your theming side. 

And as a suggestion, I would also ask you to just check out the https://getmdl.io/ which is originally made by google.

**Comment by rsercano** [comment]
@smartclash thanks for the suggestions, they look awesome, but just to catch features of current enterprise level mongodb tools, I'm spending most of my spare time to developing new features or improving existing ones, though I got a really little spare time since I'm a full time worker. I actually would like to get help from someone about design / templating.

**Comment by smartclash** [comment]
Well, if it is so, then I am ready to help you in it. @rsercano

**Comment by rsercano** [comment]
@smartclash great news thank you so much, I would like to discuss details, you can reach me from ozdemirsercan27@gmail.com

**Comment by zspine** [comment]
@rsercano Thanks for the response. institutionalization? actually sounds exciting.... lets see how it goes!

@rsercano @smartclash Yes.. agree and that would be amazing!

**Comment by rsercano** [comment]
@zspine thanks mate, that's one of the possibilities for mongoclient in near future :)

**Comment by beeman** [comment]
Another good candidate for the UI would be CoreUI: http://coreui.io/

Personally I don't think an OSS project should need to spend $1000 to buy a license, there are so many good free options out there :)

@rsercano  Let me know if you need any help re-skinning it.

**Comment by rsercano** [comment]
@beeman thanks for the suggestion, gotta check it as well :) 
But I got huge plans on mongoclient, ofc it will stay as OSS and free, inspina comes with a bunch of feature and I currently don't want to lose them, thank you so much for the help offer :)

**Comment by mattwelke** [comment]
On the topic of themes, one of my concerns is readability. I prefer dark themes, especially for my home computer, since my monitors display light themes in a way that makes it hard to make out text or subtle differences in background (for sections of a page, etc).

I also noticed that there were lots of transitions/animations in the app that made it slow to navigate around. It didn't feel very fluid. I would prefer seeing the app just instantly display what you've selected, instead of having animations etc.

Despite these concerns, I do feel that an open source cross-platform client for Mongo is a great idea, so I donated to the project. Good luck with the theming. :)
